src/SimCore/KaonPhysics.cxx

```diff
@@ -18,16 +18,22 @@ KaonPhysics::KaonPhysics(const G4String& name,
       parameters.getParameter<double>("kminus_lifetime_factor");
   k0l_lifetime_factor = parameters.getParameter<double>("k0l_lifetime_factor");
   k0s_lifetime_factor = parameters.getParameter<double>("k0s_lifetime_factor");
+  verbosity = parameters.getParameter<int>("verbosity");
 }
 void KaonPhysics::setDecayProperties(
     G4ParticleDefinition* kaon, const std::vector<double>& branching_ratios,
     double lifetime_factor) const {
-  kaon->SetPDGLifeTime(kaon->GetPDGLifeTime() * lifetime_factor);
   auto table{kaon->GetDecayTable()};
   if (!table) {
     EXCEPTION_RAISE("KaonPhysics", "Unable to get the decay table from " +
                                        kaon->GetParticleName());
   }
+  if (verbosity > 1) {
+    std::cout << "Decay details after setting branching ratios and lifetimes"
```

Comment:
Here and other places, would it be possible to use the logger instead of standard printouts? I see a use case where we use these modifications routinely and would only want to see details at debug or info level.

Reply:
Sure, although I'm not familiar with how to do that with things in LDMX-sw that aren't processors

Reply:
Yep,  was straight-forward to do although the enableLogging macro didn't play nicely by default with the namespace I was in so had to do 
```C++
  mutable framework::logging::logger theLog_{
      framework::logging::makeLogger("KaonPhysics")};
```
directly. Not the end of the world I guess